dialogue-core/src/main/java/com/palantir/dialogue/core/LimitedChannel.java

```diff
@@ -29,7 +29,13 @@
  * Limited channels must limit exclusively based on the state of the {@link com.palantir.dialogue.Channel}, not
  * the {@link Endpoint} or {@link Request} arguments, otherwise the caller (generally a {@link QueuedChannel})
  * may prevent <i>all</i> requests from proceeding.
+ * Implementations should support skipping the limit checks, while still tracking usage appropriately.
  */
 interface LimitedChannel {
-    Optional<ListenableFuture<Response>> maybeExecute(Endpoint endpoint, Request request);
+    Optional<ListenableFuture<Response>> maybeExecute(Endpoint endpoint, Request request, SkipLimits skipLimits);
+
+    enum SkipLimits {
+        Yes,
+        No
+    }
```

Comment:
Perhaps something along the lines of:
```suggestion
    enum LimitEnforcement {
        DEFAULT_ENABLED,
        DANGEROUS_BYPASS_LIMITS
    }
```

Reply:
+1 will do manually cause need to do in an IDE.